tasks/main-blivet.yml

```diff
@@ -161,6 +161,33 @@
     daemon_reload: true
   when: blivet_output['mounts']
 
+- name: Set directory owner
```

Comment:
can this be done before the daemon_reload, or must it be done after?  I'd like to reduce the time between the mount point being created and the ownership/permissions being set, to reduce security race conditions

Reply:
Moving it before daemon_reload does not seem to cause any trouble so far. Moved.